symbolic-symcache/src/format.rs

```diff
@@ -180,6 +187,8 @@ pub struct FuncRecord {
     /// High bits of the address.
     pub addr_high: u16,
     /// The length of the function. A value of 0xffff indicates that the size is unknown.
+    /// We cannot cache any useful information for functions containing no instructions, so
+    /// the length is always positive.
```

Comment:
As you're changing these things anyway, I think proper rustdoc style is:
```
Single summary line with full stop.
<blank line here>
More description.
```
this would affect lots of other doc comments you touched as well